class-workflow.php

```diff
@@ -118,7 +118,7 @@ private function load_modules() {
 	 * @uses add_action() To add various actions
 	 */
 	private function setup_actions() {
-		add_action( 'init', [ $this, 'action_init' ] );
+		add_action( 'init', [ $this, 'action_init' ], 8 );
```

Comment:
Credit to @alecgeatches for helping with this, as I kept running into an inception situation.